### ISSUE TITLE
extension doesn't stay enabled

### ISSUE THREAD

**Issue by joshuasm** [ToastyStoemp/Hack.Chat-Enhancement-kit#2]
in chrome the extension does not stay enabled for some reason

**Comment by ToastyStoemp** [comment]
Yeah, cause the extension is not yet in the chrome market, chrome thinks it got installed without your approval. Removing it from chrome and reinstalling is a temp fix
